README.Rmd

```diff
@@ -27,22 +27,23 @@ knitr::opts_chunk$set(
 
 <!-- badges: end -->
 
-Lightweight web apps for testing.
+Lightweight fake web apps for testing.
 Built using the [civetweb](https://github.com/civetweb/civetweb) embedded web server.
 
 ## Features
 
--   Complete web app framework, define handlers for HTTP request in R.
+-   Complete web app framework, define handlers for HTTP requests in R.
+-   Write your own app for your custom test cases; our use app similar to the <https://httpbin.org> API, so if you are writing an HTTP client (httr, curl, crul) you don't need to write your own web app.
```

Comment:
I think httpbin is handy for other packages as well. This is mostly web clients, but not only: https://github.com/search?q=org%3Acran+httpbin&type=Code

Reply:
I rephrased :-)